onebusaway-android/src/main/res/layout/trip_plan_advanced_settings_dialog.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This dialog styling doesn't match the material design guidelines:

![image](https://cloud.githubusercontent.com/assets/928045/16086516/c2d6adc4-32ed-11e6-92fa-f11db61be1c8.png)

Background shouldn't be the theme color, and the button, combo box, text boxes, and fields should all use the default material design styling.  In many cases you should be able to get the default material design imply by removing the color attributes.  See https://material.google.com/components/dialogs.html#dialogs-specs for examples.